first-aid.lic

```diff
@@ -8,24 +8,46 @@ class FirstAid
   include DRC
 
   def initialize
+    settings = get_settings
     @chart_data = get_data('anatomy-charts').first_aid_charts
-
-    train_first_aid
+	@textbook = settings.textbook
```

Comment:
You don't need to make this an instance variable (or even a variable) if it's only going to be used once.